examples/car_control_complementarity/car_control_complementarity.py

```diff
@@ -0,0 +1,132 @@
+# This is an example use of path complementarities to enforce not braking and accelerating
+# at the same time.
+
+import nosnoc
+import casadi as ca
+import numpy as np
+import matplotlib.pyplot as plt
+
+X0 = np.array([0, 0])
+X_TARGET = np.array([500, 0])
+
+def car_model():
+    q = ca.SX.sym('q')
+    v = ca.SX.sym('v')
+    x = ca.vertcat(q, v)
+
+    u = ca.SX.sym('u',2)
+    lbu = np.zeros((2,))
+    ubu = np.ones((2,))
+
+    k1 = 5
+    k2 = 3
+    kb = 5
+
+    j_a = 1
+    j_b = 1
+
+    A = np.array([
+        [0, 1],
+        [0, 0]
+    ])
+
+    B1 = np.array([
+        [0,0],
+        [k1,-kb]
+        ])
+    B2 = np.array([
+        [0,0],
+        [k2,-kb]
+        ])
+
+    f_1 = A@x + B1@u
+    f_2 = A@x + B2@u
+
+    F = [ca.horzcat(f_1, f_2)]
+
+    c = [v-15]
+    S = [np.array([[-1],[1]])]
+
+    g_terminal = x - X_TARGET
+
+    f_q = j_a*u[0]**2 + j_b*u[1]**2
+
+    g_path_comp = ca.horzcat(u[0], u[1])
```

Comment:
I would have thought a tuple interface is more intuitive to not deal with the shapes.

Reply:
It would also mean that if we wanted nice errors we would have to check shapes of the two tuple members. This way `horzcat` sort of does that for us. I could see it either way though.